index.html

```diff
@@ -18,7 +18,111 @@
       <h1>Quiz App</h1>
     </header>
     <main>
-      <article class="Card">
```

Comment:
All classes should start with a small letter, rest LGTM!
![](https://media.giphy.com/media/0eESeHpic221YgThM2/giphy.gif)